frontend/src/components/Home/Autocomplete.tsx

```diff
@@ -179,7 +185,16 @@ export default function Autocomplete() {
         placeholder="Search by landlord or building address"
         className={text}
         variant="outlined"
-        onKeyDown={(event) => (event.key === 'ArrowDown' ? setFocus(true) : setFocus(false))}
+        onKeyDown={(event) => {
```

Comment:
What do you think about moving this code to a separate function, similar to `handleListKeyDown`? I think that might be cleaner. Also do we need the `else` statement?